config/splat.us.weapon.yaml

```diff
@@ -95,8 +95,7 @@ segments:
     subalign: 4
     subsegments:
       - [0x19040, animset, w_003_1]
-      - [0x19884, animset, w_003_2]
```

Comment:
I think Xeeynamo needs to comment on these since I think this is being handled by the asset manager now?

Reply:
I took the second animset out for files where it's not used.